docs/docs/deploy/deploy-rasa-pro-services.mdx

```diff
@@ -119,6 +119,7 @@ environment variables:
 | `RASA_PRO_LICENSE`        | **Required**. The license key for Rasa Pro Services.                                                                  |                    |
 | `KAFKA_BROKER_ADDRESS`    | **Required**. The address of the Kafka broker.                                                                        |                    |
 | `KAFKA_TOPIC`             | **Required**. The topic Rasa Plus publishes events to and Rasa Pro consumes from.                                     | `rasa_core_events` |
+| `LOGGING_LEVEL`           | Set the log level of the application. Valid levels are DEBUG, INFO, WARNING, ERROR, CRITICAL.                         | `INFO`             |
```

Comment:
We should mention from which version is this available.

Reply:
Oh then let me do a release for rasa-pro-services

Reply:
in this case I'd recommend backporting the change to `3.0.x` and making a micro release afterwards.

Reply:
Should I update on 3.1.x and 3.0.x both?

Reply:
If both versions are maintained then that would be a great idea 👍🏻

Reply:
I've mentioned 3.0.2 on it, when 3.1.x is released I'll also mention that